userspace/libsinsp/container.h

```diff
@@ -55,11 +106,19 @@ class sinsp_container_info
 	{
 	}
 
+	static void parse_json_mounts(const Json::Value &mnt_obj, vector<container_mount_info> &mounts);
+
+	container_mount_info *mount_by_idx(uint32_t idx);
+	container_mount_info *mount_by_source(std::string &source);
+	container_mount_info *mount_by_dest(std::string &dest);
+
 	string m_id;
 	sinsp_container_type m_type;
 	string m_name;
 	string m_image;
 	uint32_t m_container_ip;
+	bool m_privileged;
```

Comment:
For non-docker containers, the filtercheck always returns NULL. For docker containers, m_privileged should be set while parsing the results of the GET /containers/XXX/json response.I think the result always has a Privileged field.

Regardless, I should initialize it to false just to make sure. I've got that fix in https://github.com/draios/sysdig/pull/658.